packages/tasit-action/src/NFT.test.js

```diff
@@ -35,4 +90,69 @@ describe("TasitAction.NFT", () => {
     expect(name).to.exist;
     expect(name).to.equal("Full NFT");
   });
+
+  describe("ERC721 functions", () => {
+    const tokenId = 1;
+
+    beforeEach("should mint one token to owner", async () => {
+      const balanceBefore = await fullNFT.balanceOf(owner.address);
+      expect(toBN(balanceBefore)).to.be.bignumber.equal(toBN(0));
+
+      txSubscription = fullNFT.mint(owner.address, tokenId);
+      await txSubscription.waitForNonceToUpdate();
+
+      const balanceAfter = await fullNFT.balanceOf(owner.address);
+      expect(toBN(balanceAfter)).to.be.bignumber.equal(toBN(1));
+    });
+
+    it("should transfer an owned token", async () => {
+      const balanceBefore = await fullNFT.balanceOf(ana.address);
+      expect(toBN(balanceBefore)).to.be.bignumber.equal(toBN(0));
+
+      contractSubscription = fullNFT.subscribe();
+
+      txSubscription = fullNFT.transferFrom(
+        owner.address,
+        ana.address,
+        tokenId
+      );
+
+      const event = await new Promise(function(resolve, reject) {
+        contractSubscription.on("Transfer", message => {
+          const { data } = message;
+          const { args } = data;
+          resolve(args);
+        });
+      });
+
+      expect(event.from).to.equal(owner.address);
```

Comment:
Transfering from ana to bob might be a cleaner example, so the owner transferring is more like airdropping than a normal transfer

Reply:
I'm resolving this because you addressed it in your latest changes